docs/using-the-jdbc-driver/using-plugins/UsingTheFederatedAuthPlugin.md

```diff
@@ -0,0 +1,54 @@
+# Federated Authentication Connection Plugin (Proof of Concept)
+
+## Assumptions
+- AWS Aurora Database.
+- IAM user set up to access database
+- IAM role is set up for access to database.
+- ADFS is set up as a SAML Identity Provider for IAM.
+
+## How to use it
+- Add `federatedAuth`to the `wrapperPlugins` parameter. 
```

Comment:
Can you please add a simple example of what the `parameter` would look like so users can just copy paste it into their program?

In general let's try to make our documentation as "copy-pastable" as possible to reduce friction during integration.

Reply:
Done!